anaconda_project/project.py

```diff
@@ -1573,3 +1585,18 @@ def use_changes_without_saving(self):
         """
         self.project_file.use_changes_without_saving()
         self.lock_file.use_changes_without_saving()
+
+    @property
+    def bootstrap_env_prefix(self):
+        """Fullpath to bootstrap environment prefix."""
+        return join(self._directory_path, 'envs', 'bootstrap-env')
+
+    @property
+    def is_running_in_bootstrap_env(self):
+        """Return True if anaconda-project is running inside a project bootstrap env False otherwise."""
+        return os.environ['CONDA_PREFIX'] == self.bootstrap_env_prefix
+
+    @property
+    def has_bootstrap_env_spec(self):
```

Comment:
This does **not** feel like a property, this is answering a question... like a function.

`bootstrap_env_spec` could return the spec or None but `has_bootstrap_env_spec` feels like a function

Reply:
Maybe... yeah. Need to check the code again to reply on this one.